css/utilities/margin.css

```diff
@@ -0,0 +1,3 @@
+.margin-r-n1 {
+  margin-right: -1rem;
```

Comment:
it could be `-margin-r-1`

Reply:
> it could be `-margin-r-1`

```-n1``` is used for the negative value.

Reply:
Yes. classes for both negative and positive values can be a bit more identical. like `margin-r-1` and `-margin-r-1`. totally a suggestion, though.

Reply:
Thanks a lot for this suggestion, but don't you think If we will use `-margin-r-1` then it will be hard to differentiate b/w `negative (-)` and `-` used as a separator in words like `margin-r`.

Reply:
Sure. I saw tailwind following something similar. What you did is also fine.